PointOfDispensingApp/src/components/CreateLocationResourcePage.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <v-container>
+    <v-row>
+      <h2 class="font-weight-medium primary--text">Location Resources</h2>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-divider></v-divider>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <div class="font-weight-medium secondary--text">Enter one or more search parameters to retrieve the available location resources.</div>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="5">
+        <v-row no-gutters>
+          <v-col cols="4">
+            <div class="secondary--text">City</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_city"
+            ></v-text-field>
+          </v-col>
+          <v-col cols="4">
+            <div class="secondary--text">Zip Code</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_postalCode"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+      </v-col>
+      <v-col cols="1">
+        <!--blank column for spacing-->
+      </v-col>
+      <v-col cols="5">
+        <v-row no-gutters>
+          <v-col cols="4">
+            <div class="secondary--text">State</div>
+          </v-col>
+          <v-col cols="8">
+            <v-select
+              :items="stateOptions"
+              outlined
+              dense
+              v-model="search_state"
+            ></v-select>
+          </v-col>
+          <v-col cols="4">
+            <div class="secondary--text">Organization</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_organization"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row> 	
+    <v-row no-gutters>
+      <v-col cols="12">
+        <v-btn color="accent" @click="retrieveLocationResources()">
+          Search
+        </v-btn>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-divider></v-divider>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <template>
+          <v-data-table
+            @click:row="rowClick"
+            item-key="siteID"
+            single-select
+            :items="locationResourceLookupTable"
+            :headers="headers"
+            class="elevation-1"
+            :footer-props="{
+              'items-per-page-options':[5]
+            }"
+            :items-per-page="5"
+          ></v-data-table>
+        </template>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-dialog
+          v-model="createLocationResourceDialog"
+          persistent
+          max-width="70%"
+          >
+            <template v-slot:activator="{ on, attrs }">
+              <v-btn color="accent" v-bind="attrs" v-on="on">
+                Create location resource
```

Comment:
We need to stop talking about things in terms of FHIR. The admin at the site will not have an idea of why we're calling it a resource and will be confused (as I was looking at it). 

I'd just really like to see us not use "resource" in our UI

Reply:
Changed the button label to "Create Point of Dispensing Location"